src/fastga_he/models/performances/mission_vector/constants.py

```diff
@@ -6,3 +6,4 @@
 HE_SUBMODEL_EQUILIBRIUM = "submodel.performances_he.equilibrium"
 HE_SUBMODEL_ENERGY_CONSUMPTION = "submodel.performances_he.energy_consumption"
 SUBMODEL_CG_VARIATION = "submodel.performances.cg_variation"
+SUBMODEL_DELTA_M = "submodel.performances.delta_m"
```

Comment:
While I don't think we need to rename the old services, the new one should adopt the right naming convention !